package_reloader.py

```diff
@@ -60,6 +62,12 @@ def run_async(self, pkg_name=None):
         pkg_name = self.current_package_name
 
         if pkg_name:
+
+            # Clear any package loggers
+            logger = logging.getLogger(pkg_name)
+            for handler in list(logger.handlers):
+                logger.removeHandler(handler)
```

Comment:
I am not sure if I like this idea. It should be handled by the package but not APR.

Reply:
Sorry, I didn't mean this to part of the last PR.